lib/components/form-fieldset.vue

```diff
@@ -27,12 +38,15 @@
                 return this.horizontal ? ('col-sm-' + (12 - this.labelSize)) : 'col-12';
             }
         },
-        mounted() {
-            const content = this.$refs.content;
-            if (!content) {
-                return;
+        methods: {
+            target() {
```

Comment:
OK so leave it as a method, and then have `mounted()` and `updated()`call the method and update a data value, which is then referenced by the the `<label>`'s `:for`